src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -435,12 +435,6 @@ public function unload(TestCase $test): void
                     $fixture->dropConstraints($db);
                 }
             }
-
-            foreach ($fixtures as $fixture) {
-                if ($this->isFixtureSetup($configName, $fixture)) {
-                    $fixture->truncate($db);
-                }
-            }
```

Comment:
This code has been around for a looooong time. However, it doesn't appear to be impacting the testsuite of today, and wouldn't be part of the fixtures recovery behavior for when a test crashes, so lets try it out.